app/elements/socobo-recipe/socobo-recipe-details-show.html

```diff
@@ -78,19 +79,36 @@
     }
 
     /* TITLE */
+    #title_default {
```

Comment:
- more consistency
- fast find if The element has a styling
- classes for styling
- ids for Dom access